<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1262</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Estado</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Proveedor</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="provedorLine">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="numeroLine3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Numero</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="numeroLine">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Precio unitario</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="precioUnitarioLine">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="otro4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lotro4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Garantia</string>
    </property>
   </widget>
   <widget class="QLabel" name="lotro3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>210</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo</string>
    </property>
   </widget>
   <widget class="QLabel" name="lotro2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Marca</string>
    </property>
   </widget>
   <widget class="QLabel" name="lotro1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Otro Proveedor</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="otro1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Electronico</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Mobiliario</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>410</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>50</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo</string>
    </property>
   </widget>
   <widget class="QLabel" name="laberDesglosarM">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>30</y>
      <width>301</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Dado un tipo desglosar marcas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>70</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Obtener</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>160</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Obtener</string>
    </property>
   </widget>
   <widget class="QLabel" name="malEstadoTipo">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>130</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Medios en malestado desglosados por tipo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>230</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Obtener</string>
    </property>
   </widget>
   <widget class="QLabel" name="mayorMenorGarantia">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>230</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Equipo de mas y menos garantia</string>
    </property>
   </widget>
   <widget class="QLabel" name="mayorProveedor">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>290</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mayor proveedor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>290</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Obtener</string>
    </property>
   </widget>
   <widget class="QLabel" name="costoTotal">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>350</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Costo total</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>350</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Obtener</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>410</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Obtener</string>
    </property>
   </widget>
   <widget class="QLabel" name="costoPromedio">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>410</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Costo promedio</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>82</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Bien</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Regular</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>82</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Aula</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Laboratorio</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Apartamento</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Oficina</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Refrigerador</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Video</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ventilador</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Aire acondicionado</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Computadora</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Silla</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Buro</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cama</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mueble PC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mesa</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_5">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>70</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Refrigerador</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Video</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ventilador</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Aire acondicionado</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Computadora</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>82</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Sony</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sanyo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Atec</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Panda</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LG</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>460</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Obtener</string>
    </property>
   </widget>
   <widget class="QLabel" name="tiposMobiliario">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>460</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Desglosados mobiliarios por tipo</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>120</y>
      <width>651</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>190</y>
      <width>651</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>260</y>
      <width>651</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>320</y>
      <width>651</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>380</y>
      <width>651</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>440</y>
      <width>651</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1262</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
